app/src/main/java/ingsw/group1/findmyphone/activity/ContactListActivity.java

```diff
@@ -13,36 +19,50 @@
 import java.util.List;
 
 import ingsw.group1.findmyphone.R;
+import ingsw.group1.findmyphone.contacts.SMSContactRecyclerAdapter;
+import ingsw.group1.findmyphone.contacts.SMSContactSearchListener;
+import ingsw.group1.findmyphone.contacts.SMSContactSwipeCallback;
 import ingsw.group1.findmyphone.contacts.SMSContact;
 import ingsw.group1.findmyphone.contacts.SMSContactManager;
 
 /**
  * Activity for the view showing the contact list
+ * using a {@link RecyclerView} to display contacts list
+ * and a {@link SMSContactRecyclerAdapter} to populate and manage the list of contacts saved.
+ * Contacts are saved in a database managed by a {@link SMSContactManager}
+ * and viewed in alphabetical order.
  *
  * @author Giorgia Bortoletti
  */
 public class ContactListActivity extends AppCompatActivity {
 
+    private SMSContactManager contactManager;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'contactManager' could be replaced by a local variable.](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=5065092)